pkg/gofr/datasource/pubsub/nats/health.go

```diff
@@ -0,0 +1,84 @@
+package nats
+
+import (
+	"context"
+	"time"
+
+	"github.com/nats-io/nats.go"
+	"github.com/nats-io/nats.go/jetstream"
+	h "gofr.dev/pkg/gofr/health"
+)
+
+const (
+	natsBackend            = "NATS"
+	jetstreamStatusOK      = "OK"
```

Comment:
Use camel-case for `jetStream` in all the defined constants

Reply:
done